gbpservice/neutron/nsf/configurator/api/v1/controllers/controller.py

```diff
@@ -0,0 +1,181 @@
+import json
+import subprocess
+
+from neutron.common import rpc as n_rpc
+from neutron.agent.common import config
+from oslo_config import cfg
+import oslo_messaging
+from pecan import expose, request
+from pecan import rest
+
+import constants
+
+
+class Controller(rest.RestController):
+
+    """controller class for handling all the curl request"""
+
+    def __init__(self, module_name):
+        self.host = subprocess.check_output('hostname', shell=True).rstrip()
```

Comment:
Any shell command that we run should be in a separate library. Or any OS specific code should be in separate library. When configurator moves to different OS, it will be easy to port the code.